logic/block.js

```diff
@@ -382,11 +378,23 @@ Block.prototype.schema = {
  * @todo Add description for the function and the params
  */
 Block.prototype.getBytes = function(block) {
-	const size = 4 + 4 + 8 + 4 + 4 + 8 + 8 + 4 + 4 + 4 + 32 + 32 + 64;
+	const capacity =
+		4 + // version (int)
+		4 + // timestamp (int)
+		8 + // previousBlock
+		4 + // numberOfTransactions (int)
+		8 + // totalAmount (long)
+		8 + // totalFee (long)
+		8 + // reward (long)
+		4 + // payloadLength (int)
+		32 + // payloadHash
+		32 + // generatorPublicKey
+		64 + // blockSignature or unused
+		4; // unused
```

Comment:
It would even be nice to store these in variables/constants.

Reply:
Agree. But I'd prefer to to do this as a follow up step. The idea here is to make things readable without risking any functional change and without requiring major refactoring decisions. There are more follow up step to be done after this. For example decide what to do with those 4 bytes of unused capacity.

Reply:
Besides, we should check figures apply against these variables before writing to the buffer.

Reply:
@IkerAlus last 4 unused bytes might have any not clear use?

Reply:
Those 4 bytes do not make it into the result of `getBytes`. See for example

```
const ByteBuffer = require('bytebuffer');
var b = new ByteBuffer(30, true);
b.writeLong(123);
b.flip();
b.toBuffer().length; // returns 8
```

Reply:
This is a different list of numbers to what's in place currently. Can you explain the discrepancy?

Reply:
Jap, see this commit https://github.com/LiskHQ/lisk/pull/1874/commits/1ba82f540981e47a594f76da1651ebc5e040095b. A 4 was associated with int, which was wrong and an unused extra 4 was moved to the end to get it out of the way.